docs/src/increase-observability/metrics/_index.md

```diff
@@ -71,25 +71,24 @@ Burst allows your container to use additional resources when they aren't require
 
 ## Time intervals
 
-Measurements are taken for each metric _every ten seconds_ for Dedicated environments and _every 1 minute_ for Grid environments.
+Measurements are taken for each metric every 1 minute.
```

Comment:
:bulb: Do we want to update the video and the first screenshot to reflect that change as well?

Reply:
The UI isn't changed yet, so especially the video might be slightly trickier. Not sure it's worth it for something that isn't the focus of the images. :shrug: